sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -165,21 +165,23 @@ def install_instrumentation
       end
 
       def configure_span_processors
-        processors = @span_processors.empty? ? [wrapped_exporter_from_env].compact : @span_processors
+        processors = @span_processors.empty? ? wrapped_exporter_from_env.compact : @span_processors
         processors.each { |p| tracer_provider.add_span_processor(p) }
       end
 
       def wrapped_exporter_from_env
-        exporter = ENV.fetch('OTEL_TRACES_EXPORTER', 'otlp')
-        case exporter
-        when 'none' then nil
-        when 'otlp' then fetch_exporter(exporter, 'OpenTelemetry::Exporter::OTLP::Exporter')
-        when 'jaeger' then fetch_exporter(exporter, 'OpenTelemetry::Exporter::Jaeger::CollectorExporter')
-        when 'zipkin' then fetch_exporter(exporter, 'OpenTelemetry::Exporter::Zipkin::Exporter')
-        when 'console' then Trace::Export::SimpleSpanProcessor.new(Trace::Export::ConsoleSpanExporter.new)
-        else
-          OpenTelemetry.logger.warn "The #{exporter} exporter is unknown and cannot be configured, spans will not be exported"
-          nil
+        exporters = ENV.fetch('OTEL_TRACES_EXPORTER', 'otlp')
+        exporters.split(',').map do |exporter|
```

Comment:
I think it's worth guarding for spaces here - it could reasonably be classified as "user error", but it's also trivial to end up with something like `["otlp", " jaeger"]`:

```
~
❯ OTEL_TRACES_EXPORTER="otlp, jaeger" irb

irb(main):001:0> ENV.fetch('OTEL_TRACES_EXPORTER', 'otlp').split(',')
=> ["otlp", " jaeger"]   # note, extra space
irb(main):002:0>
```

Reply:
@SomalianIvan If it is not too much trouble I would also ask that you add an additional test case for removing white space.

Reply:
For sure :)